.github/workflows/cv.yml

```diff
@@ -24,7 +24,7 @@ on:
 jobs:
   commit-validation:
     name: Commit-validation (on Linux)
```

Comment:
nit: maybe we can rename it to "Commit-validation (Linux, JDK 11)" (same for the other jobs)